src/App.js

```diff
@@ -1,24 +1,95 @@
 import React from 'react';
-import logo from './logo.svg';
+import {BarChart, ScatterPlot, LineChart} from 'react-d3-components'
+import data from './assets/MOCK_DATA_150.json'
 import './App.css';
 
+/**
+ * This file is temporary! It will be modified after routing and logic are set - Nikita
+ */
+
+const style = {
+  fill: '#61dafb',
+  fillOpacity: '0.125'
+}
+
 function App() {
+  let d9 = data.map(s => {
+    return {
+      x: parseFloat(s.current_salary.slice(1)),
+      y: parseFloat(s.previous_salary.slice(1)),
+    }
+  })
+  let dF = d9
+
+  dF.sort((a,b)=> a.x - b.x)
+
+  console.log("dF",dF)
+
+  let dP = {
+    label: 'somethingC',
+    values: [...dF]
+  }
+
+  let d0 = data.map(s => {
+    return {
+      x: parseFloat(s.current_salary.slice(1)),
+      y: parseFloat(s.previous_salary.slice(1)),
+    }
+  })
+
+  let dG = d0
+
+  dG.sort((a,b) => a.x - b.x)
+
+  console.log("dG",dG)
```

Comment:
This one also.

Reply:
removed console.log but need sort function as part of what we're demonstrating.